scripts/zkai_attention_kv_d16_quantized_softmax_receipt_gate.py

```
@@ -0,0 +1,573 @@
+    source = source_input()
+    envelope = fused_envelope()
+    # This validates the previous fused proof artifact, including the native CLI proof verifier.
+    fused_gate.run_gate()
+
+    receipt = build_base_receipt(source)
+    mutation_results = []
+    for name, mutated_receipt, mutated_source, mutated_envelope, run_native in mutation_cases(receipt, source, envelope):
+        try:
+            validate_receipt(mutated_receipt, mutated_source, mutated_envelope, run_native=run_native)
+        except Exception as err:  # noqa: BLE001 - gate records exact rejection surface.
+            mutation_results.append({"name": name, "rejected": True, "error": str(err)})
```

Ruff (0.15.12)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)